soc/xtensa/esp32/CMakeLists.txt

```
@@ -5,3 +5,77 @@ zephyr_sources(
   esp32-mp.c
   loader.c
   )
+
+  if(CONFIG_BOOTLOADER_ESP_IDF)
```

minor nit.
The indentation of the `if()` is of.
```suggestion
if(CONFIG_BOOTLOADER_ESP_IDF)
```